2-farm/lessons/1-predict-plant-growth/README.md

```diff
@@ -264,7 +264,11 @@ For these, find if there are sensors that can measure them. What about actuators
 Read more on digital agriculture on the [Digital Agriculture Wikipedia page](https://wikipedia.org/wiki/Digital_agriculture). Also read more about precision agriculture the [Precision Agriculture Wikipedia page](https://wikipedia.org/wiki/Precision_agriculture).
 
 The full growing degree days calculation is more complicated than the simplified one given here. Read more about the more complicated equation and how to deal with temperatures below the baseline on the [Growing Degree Day Wikipedia page](https://wikipedia.org/wiki/Growing_degree-day).
-
+<br>
```

Comment:
Can you use markdown please instead of inline HTML

Reply:
@jimbobbennett most experienced GitHub Markdown users use HTML links for URLs in order for the link to open in a NEW tab when users click. Plain URL in Markdown would replace the same tab with the destination page. There are pluses and minuses to each approach. Adding HTML has the potential for more errors (such as double quotes).
Since you're the owner of the site, please specify your preference for this convention in a CONTRIBUTING page.